examples/notebook/tensorflow/alexnet_mnist/run_in_intel_devcloud.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 echo "Enable Conda Env."
-source ${ONEAPI_ROOT}/intelpython/python3.9/etc/profile.d/conda.sh
+source /opt/intel/oneapi/setvars.sh
```

Comment:
In devcloud, this path is not working.
Please use "${ONEAPI_ROOT}/intelpython/python3.9/etc/profile.d/conda.sh" if verify is passed.

Reply:
Hi,
After verifying, I implemented the changes mentioned above.

Reply:
OK! please ignore my comments.